### PR TITLE
feat(i18n): Add a functionality for operators translation

### PR DESCRIPTION
**Pull request by IraErshova**
Create operators service
Add translation for _do_ operator
Update unit tests

### CODE REVIEW

**Comment by valorkin** [comment]
@IraErshova please don't move EN version into `en` folder
then you will have to move all operator docs
and this will be not cool at all

File: src/app/core/services/operators.service.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  constructor(private languageService: LanguageService) {
+    this.currentLang = this.languageService.getCurrentLang();
+
+    this.initOperators(this.currentLang.code);
+  }
+
+  initOperators(code: string): void {
+    switch (code) {
+      case 'en':
```
Comment (valorkin) [inline src/app/core/services/operators.service.ts]: in this case you should define 'en' as a last case with fallback to default

File: src/app/operators/components/operator/operator.component.ts [hunk 2]
```diff
@@ -33,30 +28,34 @@ export class OperatorComponent implements OnInit {
...
-            ? this.operator.shortDescription.description
-            : ''
-        });
+    this._operatorsService.getOperators().subscribe(data => {
+      this.operators = data;
+      this.operators.forEach((op: OperatorDoc) => {
+        this.operatorsMap.set(op.name, op);
       });
+
+      this._activatedRoute.params
```
Comment (valorkin) [inline src/app/operators/components/operator/operator.component.ts]: unsubscribe on component destroy

File: src/tsconfig.app.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../out-tsc/app",
     "baseUrl": "./",
-    "module": "es2015",
+    "module": "esnext",
```
Comment (valorkin) [inline src/tsconfig.app.json]: is it completely necessary?

Comment (IraErshova) [reply src/tsconfig.app.json]: @valorkin  dynamic import won't work without this

Comment (valorkin) [reply src/tsconfig.app.json]: my mistake

**Pull request merged**